app/2.0/docs/tools/polymer-cli.md

```diff
@@ -303,20 +303,43 @@ Open up a page other than the default `index.html` in a specific browser
 
 ### `polymer lint` {#lint}
 
-Lints element(s)..
+Analyze your project for syntax errors, missing imports, bad databinding expressions and more.
```

Comment:
Is it worth adding a note here about what `polymer lint` _doesn't_ do? 

As I understand it, it doesn't catch general JavaScript issues, so people might want to supplement it with a general-purpose linter like ESLint or JSHint.